doc/API.rst.txt

```diff
@@ -106,10 +106,6 @@ change is made.
 * `scipy.io`
 
   - `scipy.io.arff`
```

Comment:
This is a little unfortunate, but yes `arff` and `wavfile` have docs and are public submodules even though they are pretty much empty. Let's leave them alone indeed.